tools/additional-per-crate-checks.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+
+if [[ $# -ne 1 ]]; then
```

Comment:
I must be missing something, but I don't see this actually being run in CI?

Reply:
It's at the very end of ci.yaml, as part of runtime tests.

Reply:
Oh! Github didn't expand that file and I thought it was deleted. Didn't realize it was edited 🤕